### PR TITLE
[UI] Edit backup offering

### PR DESCRIPTION
**Pull request by SadiJr**
### Description

This PR allows users to edit a name and description of a Backup Offering

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

### Screenshots (if appropriate):
image
image
image
image
image

### How Has This Been Tested?
I tested in a local lab.
1. I imported a Backup Offering;
2. I edited the name of this offering;
3. I edited the description of this offering.
4. I checked, in UI and DB if changes are applied.

### CODE REVIEW

File: ui/src/config/section/offering.js [hunk 1]
```diff
@@ -215,6 +215,15 @@ export default {
         listView: true,
         popup: true,
         component: () => import('@/views/offering/ImportBackupOffering.vue')
+      }, {
+        api: 'updateBackupOffering',
+        icon: 'edit',
+        label: 'label.edit',
+        dataView: true,
+        groupAction: true,
```
Comment (rhtyd) [inline ui/src/config/section/offering.js:223]: You don't want groupable action on edit/updating offering, otherwise lgtm

Comment (utchoang) [reply ui/src/config/section/offering.js:223]: @SadiJr same thinking, don't set `groupAction` for edit/update

**Comment by nvazquez** [comment]
@blueorangutan ui

File: ui/src/config/section/offering.js (hunk 1 above)
Comment (SadiJr) [reply ui/src/config/section/offering.js]: @rhtyd @utchoang Done, thanks for the suggestion.

**Comment by nvazquez** [comment]
@blueorangutan ui

**Review by nvazquez — APPROVED** [review]
LGTM - manually tested using the dummy provider

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Pull request merged**